### PR TITLE
feat: 🎸 Add support for spec range 5.3.x

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

Updates supported spec version and node version to 5.3.x 

### Breaking Changes

NA

### JIRA Link

DA-571

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/utils/internal.ts [hunk 1]
```diff
@@ -1316,7 +1318,7 @@ function handleSpecVersionResponse(
     return false;
   }
 
-  if (!satisfies(specVersionAsSemver, SUPPORTED_SPEC_VERSION_RANGE)) {
+  if (gtr(specVersionAsSemver, SUPPORTED_SPEC_VERSION_RANGE)) {
```
Comment (polymath-eric) [inline src/utils/internal.ts:1321]: I think there should another be a "less than" check for the minimum version that will work.

File: src/utils/constants.ts [hunk 2]
```diff
@@ -109,12 +110,18 @@ export const ROOT_TYPES = rootTypes;
 /**
  * The Polymesh RPC node version range that is compatible with this version of the SDK
  */
-export const SUPPORTED_NODE_VERSION_RANGE = '5.2.0';
+export const SUPPORTED_NODE_VERSION_RANGE = '5.3.0';
```
Comment (polymath-eric) [inline src/utils/constants.ts:113]: I think we should have two variables, `MAX_SUPPORTED_NODE_VERSION` and `MIN_SUPPORTED_NODE_VERSION`.

For the first 5.3 release we leave MIN at 5.2 and set MAX to 5.3. Then when we add NFT features we bump the MIN to 5.3.

**Review by polymath-eric — COMMENTED** [review]
I think we should do a minimum check as well + add a min variable.

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**